TeapotTests/TeapotTests.swift

```diff
@@ -144,21 +140,24 @@ class TeapotTests: XCTestCase {
     }
 
     func testImage() {
-        // http://icons.iconarchive.com 
-        // /icons/martz90/circle/512/app-draw-icon.png
         let expectation = self.expectation(description: "GetImage")
         let url = URL(string: "http://icons.iconarchive.com/icons/martz90/circle/512/app-draw-icon.png")!
 
         Teapot(baseURL: url).get() { (result: NetworkImageResult) in
             switch result {
             case .success(let image, let response):
-                let localImage = Bundle(for: TeapotTests.self).image(forResource: NSImage.Name(rawValue: "app-draw-icon"))!
+                guard let localImage = Bundle(for: TeapotTests.self).image(forResource: NSImage.Name(rawValue: "app-draw-icon")), let tiff = localImage.tiffRepresentation else {
```

Comment:
I find having the local constant declared in the middle of a long line a bit hard to follow, compared to breaking it down line-by-line. What do you think?

Reply:
I agree but I was following our general coding standards for this. I'm definitely 👍  to break it up into two lines.

Reply:
We have that in the guidelines? 🙈 PR time there I guess hehe